site/src/sphinx/server-grpc.rst

```diff
@@ -110,7 +116,33 @@ to just pass ``GrpcSerializationFormats.values()``.
 
 The server will support both native gRPC_ and gRPC-Web_ from the same endpoint. Use the unofficial
 gRPC-Web-Client_ to access the service from a browser. gRPC-Web_ does not support RPC methods with streaming
-requests.
+requests. gRPC-Web-Client_ might first send ``preflighted`` requests by the HTTP ``OPTIONS`` method, in order
```

Comment:
I think this only happens when the request is actually cross-origin (grpc-web-client is just using browser APIs which deal with cors transparently I guess)